src/Service/Lambda/README.md

```diff
@@ -10,3 +10,29 @@ An API client for Lambda.
 ```cli
 composer require async-aws/lambda
 ```
+
+## Usage
+
+````php
+$client = new LambdaClient([
+    Configuration::OPTION_REGION => 'us-east-1',
+    Configuration::OPTION_ACCESS_KEY_ID => 'Access key ID',
+    Configuration::OPTION_SECRET_ACCESS_KEY => 'Access key Secret',
+]);
+
+$input = new InvocationRequest([
+    'FunctionName' => 'app-dev-hello_world', // Find it here: https://console.aws.amazon.com/lambda/home?region=us-east-1#/functions
+    'Payload' => '{"name": "async-aws/lambda"}',
+]);
+
+$result = $client->invoke($input);
+
+try {
+    $result->resolve();
+
+} catch (NetworkException|HttpException $e) {
```

Comment:
This is not compatible with the lowest PHP supported by this library

Reply:
👍 my bad